tools/build-tools/data/layerInfo.json

```diff
@@ -164,14 +164,16 @@
             },
             "Framework": {
                 "packages": [
-                    "@fluid-internal/client-api"
+                    "@fluid-internal/client-api",
+                    "@fluid-experimental/fluid-main"
                 ],
                 "dirs": [
                     "packages/framework/"
                 ],
                 "deps": [
                     "Hosts",
-                    "Runtime"
+                    "Runtime",
+                    "Server-Tools"
```

Comment:
Do we need to add Server-Tools?

Reply:
I had added that when tinylicious was being exported...i'll remove